app/views/membership_application_mailer/new_application.html.haml

```diff
@@ -0,0 +1,6 @@
+Hi #{@admin.first_name},
+
+%p A new application has been submitted by #{@membership_application.user_full_name}, with company number: #{@membership_application.company_number} and company id: #{@membership_application.company_id}.
+
+%br Regards,
+%em= ENV['SHF_ADMIN_EMAIL']
```

Comment:
@RobertCram will have a look at the translations and ShfMailer templates. Am I right that we are using `@membership_application.user_full_name` now?